sources/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maven:3.8.7-eclipse-temurin-17-alpine AS build
+COPY src /home/app/src
+COPY pom.xml /home/app
+RUN mvn -f /home/app/pom.xml package -DskipTests
```

Comment:
Having the version number in the JAR name makes this a bit difficult to maintain. But adding `-Dquarkus.package.output-name=jitaccess` should fix this.